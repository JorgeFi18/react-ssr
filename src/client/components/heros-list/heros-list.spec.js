import React from 'react';
import renderer from 'react-test-renderer';
import HerosList from './heros-list.component';

describe('Heros List component', () => {

    it('Should match with snapshot', () => {
        const component = renderer.create(
            <HerosList heros={[]} />
        )

        const tree = component.toJSON();

        expect(tree).toMatchSnapshot();
    });
});