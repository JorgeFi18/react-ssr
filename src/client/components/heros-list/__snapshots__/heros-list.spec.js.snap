// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heros List component Should match with snapshot 1`] = `
<div
  className="heros-list"
/>
`;
